wisembly/amqp-bundle/1.3/etc/packages/amqp.yaml

```diff
@@ -0,0 +1,7 @@
+wisembly_amqp:
```

Comment:
We never add blank lines in the config ... and in this case the config is super small. I think we should remove those blank lines.